packages/containers/src/List/List.container.js

```diff
@@ -97,7 +98,7 @@ class List extends React.Component {
 					api.action.getActionCreatorFunction(
 						this.context,
 						this.props.actions.title,
-					)(p.id),
+					)(e, p, this.context),
```

Comment:
why passing again the context when the action creator factory was already given it ?